Library/Formula/mtr.rb

```diff
@@ -36,8 +36,15 @@ def install
   end
 
   def caveats; <<-EOS.undent
-    mtr requires root privileges so you will need to run `sudo mtr`.
-    You should be certain that you trust any software you grant root privileges.
+    mtr requires superuser privileges for raw sockets access. You can either
+    run the program via `sudo`, or change its ownership to root and set the
+    setuid bit:
+
+      sudo chown root:wheel #{sbin}/mtr
```

Comment:
Deleting a file only requires write permission to the containing directory, the permissions on the file itself don't matter.